.github/workflows/build-ectrans4py.yml

```diff
@@ -0,0 +1,121 @@
+on:
+
+  # Trigger the workflow on all pushes, except on tag creation
+  push:
+    branches:
+    - '**'
+    tags-ignore:
+    - '**'
+
+  # Trigger the workflow on all pull requests
+  pull_request: ~
+
+  # Allow workflow to be dispatched on demand
+  workflow_dispatch: ~
+
+env:
+  ECTRANS_TOOLS: ${{ github.workspace }}/ectrans/.github/tools
+  CTEST_PARALLEL_LEVEL: 1
+  CACHE_SUFFIX: v1        # Increase to force new cache to be created
+
+jobs:
+  ci:
+    name: ci
+
+    strategy:
+      fail-fast: false    # false: try to complete all jobs
+
+      matrix:
+        build_type: [Release,Debug]
+        name:
+          - linux gnu-13
+          - macos
+
+        include:
+
+          - name: linux gnu-13
+            os: ubuntu-24.04
+            compiler: gnu-13
+            compiler_cc: gcc-13
+            compiler_cxx: g++-13
+            compiler_fc: gfortran-13
+            ctest_options: -E memory
+            caching: true
+
+          - name: macos
+            # Xcode compiler requires empty environment variables, so we pass null (~) here
+            os: macos-13
+            compiler: clang-14
+            compiler_cc: ~
+            compiler_cxx: ~
+            compiler_fc: gfortran-13
+            caching: true
+
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Checkout ecBuild
+      uses: actions/checkout@v4
+      with:
+        repository: ecmwf/ecbuild
+        path: ecbuild
+
+    - name: Checkout FIAT
+      uses: actions/checkout@v4
+      with:
+        repository: ecmwf-ifs/fiat
+        path: fiat
+
+    - name: Checkout ecTrans
+      uses: actions/checkout@v4
+      with:
+        path: ectrans
+
+    - name: Environment
+      run:  |
+        echo "DEPS_DIR=${{ runner.temp }}/deps" >> $GITHUB_ENV
+        echo "CC=${{ matrix.compiler_cc }}"     >> $GITHUB_ENV
+        echo "CXX=${{ matrix.compiler_cxx }}"   >> $GITHUB_ENV
+        echo "FC=${{ matrix.compiler_fc }}"     >> $GITHUB_ENV
+
+        if [[ "${{ matrix.os }}" =~ macos ]]; then
+          brew install ninja
+        else
+          sudo apt-get update
+          sudo apt-get install ninja-build
+        fi
+
+        printenv
+
+    - name: Install FIAT
+      run:  |
+        cmake -B $GITHUB_WORKSPACE/fiat/build -S $GITHUB_WORKSPACE/fiat
+        cmake --build $GITHUB_WORKSPACE/fiat/build
+        echo "fiat_ROOT=$GITHUB_WORKSPACE/fiat/build" >> $GITHUB_ENV
+
+    - name: Install FFTW
+      shell: bash -eux {0}
+      run: |
+        ${ECTRANS_TOOLS}/install-fftw.sh --version 3.3.10 --with-single --prefix ${DEPS_DIR}/fftw
+        echo "FFTW_ROOT=${DEPS_DIR}/fftw" >> $GITHUB_ENV
+
+    - name: Install OpenBLAS
+      shell: bash -eux {0}
+      run: ${ECTRANS_TOOLS}/install-openblas.sh
+
+    - name: Build ectrans4py
+      run:  |
+        python3 -m venv venv
+        . venv/bin/activate
+        pip install build
+        python -m build --wheel ectrans -o $GITHUB_WORKSPACE
+        pip install ectrans4py*
```

Comment:
Not very failiar with github actions, but I guess this is supposed to install the wheel that has just been built on `$GITHUB_WORKSPACE`, right ?

Reply:
Exactly. Every command in `steps` is executed in `$GITHUB_WORKSPACE` which here has the value `home/runner/work/ectrans/ectrans`.